docs/integration_setup_failures.md

```diff
@@ -4,13 +4,11 @@ title: "Handling setup failures"
 
 Your integration may not be able to be set up for a variety of reasons. The most common cases are because the device or service is offline or the credentials are no longer valid. Your integration must retry setup so it can recover as soon as reasonably possible when the device or service is back online without the user having to restart Home Assistant.
 
-## Handling offline or unavailable devices and services
+## Config entry not ready
 
-### Integrations using `async_setup_entry`
+if the integration is unable to make a connection to the service, it is safe to assume that this is a temporary issue. In that case, a user can raise `ConfigEntryNotReady` exception from `async_setup_entry` in the integration's `__init__.py`, and Home Assistant will automatically take care of retrying set up later.
 
-Raise the `ConfigEntryNotReady` exception from `async_setup_entry` in the integration's `__init__.py`, and Home Assistant will automatically take care of retrying set up later. To avoid doubt, raising `ConfigEntryNotReady` in a platform's `async_setup_entry` is ineffective because it is too late to be caught by the config entry setup.
-
-#### Example
+You should not raise `ConfigEntryNotReady` in a platform's `async_setup_entry` is ineffective because it is too late to be caught by the config entry setup.
```

Comment:
```suggestion
You should not raise `ConfigEntryNotReady` in a platform's `async_setup_entry` because it is too late to be caught by the config entry setup.
```